connector/connect/common/pom.xml

```diff
@@ -59,6 +59,12 @@
             <groupId>io.grpc</groupId>
             <artifactId>grpc-netty</artifactId>
             <version>${io.grpc.version}</version>
+            <exclusions>
```

Comment:
the exclusion is not required, pin version in `dependencyManagement` is sufficient